src/App.jsx

```diff
@@ -1,44 +1,28 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { tabs } from './constants/constants';
+import { Tabs } from './components/Tabs';
+import { getActiveTabTitle } from './helpers/getActiveTabTitle';
 
-export const tabs = [
-  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-];
+const startRendTabsId = tabs[0].id;
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">Selected tab is Tab 1</h1>
+export const App = () => {
+  const [tabActiveId, setTabActiveId] = useState(startRendTabsId);
+  const tabActiveIdVar = tabActiveId;
+  const activeTabTitle = getActiveTabTitle(tabs, tabActiveIdVar);
 
-    <div data-cy="TabsComponent">
-      <div className="tabs is-boxed">
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">
-              Tab 1
-            </a>
-          </li>
-
-          <li data-cy="Tab">
-            <a href="#tab-2" data-cy="TabLink">
-              Tab 2
-            </a>
-          </li>
-
-          <li data-cy="Tab">
-            <a href="#tab-3" data-cy="TabLink">
-              Tab 3
-            </a>
-          </li>
-        </ul>
-      </div>
-
-      <div className="block" data-cy="TabContent">
-        Some text 1
-      </div>
+  return (
+    <div className="section">
+      <h1 className="title">Selected tab is {activeTabTitle}</h1>
+      <Tabs
+        tabs={tabs}
+        tabActiveIdVar={tabActiveIdVar}
+        changeTabFromTabs={tabId => {
+          setTabActiveId(tabId);
+        }}
+      />
```

Comment:
you really don't need to pass all tabs and active tab id, just pass it as was mentioned in requirements:
![image](https://github.com/mate-academy/react_tabs-js/assets/102289654/de43bc59-8b8d-418f-a846-29f2aac7241b)